File: problem_solving_for_beginners/array_problems/python/min_and_max.py
#!/usr/bin/python3
"""Find min and max in elements of array"""
arr = []
n = int(input("Enter a size of array: "))

for i in range(n):
    i = int(input("Enter element of array: "))
    arr.append(i)

#write your code here
max = arr[0]
min = arr[0]

for i in arr:
    if i > max:
        max = i
        
    if i < min:
        min = i

print(f"Max = {max}")
print(f"Min = {min}")

File: problem_solving_for_beginners/array_problems/c-programming/min_and_max.c
#include<stdio.h>
/**
 * main - Find min and maxelements in array
 *
 * Return: 0 Always (success)
 */
int main(void)
{
    int arr[10], n, i, max, min;
    printf("Enter the size of array: ");
    scanf("%d", &n);

    for(i = 0; i < n; i++)
    {
        printf("Enter the size of array: ");
        scanf("%d", &arr[i]);
    }

    //Write your code here
    max = arr[0];
    min = arr[0];
    
    for (i = 0; i < n; i++)
    {
        if (arr[i] > max)
            max = arr[i];
    }
    
    for (i = 0; i < n; i++)
    {
        if (arr[i] < min)
            min = arr[i];
    }
    
    printf("Max = %d\nMin = %d\n", max, min);
  
    return 0;
}